pilot/pkg/autoregistration/controller.go

```diff
@@ -477,7 +478,7 @@ func (c *Controller) shouldCleanupEntry(wle config.Config) bool {
 	return true
 }
 
-func (c *Controller) cleanupEntry(wle config.Config) {
+func (c *Controller) cleanupEntry(wle config.Config, periodic bool) {
```

Comment:
What does `periodic` do

Reply:
It was just added for logging to indicate how the WLE is cleaned up i.e is it via DelayedQueue or via Periodic cleanup

Reply:
Yes.

Reply:
LG